app/models/miq_provision_virt_workflow.rb

```diff
@@ -395,26 +395,26 @@ def allowed_cat_entries(options)
   end
 
   def allowed_vlans(options = {})
-    @allowed_vlan_cache ||= available_vlans(options)
+    @allowed_vlan_cache ||= available_vlans_and_hosts(options)[0]
     filter_by_tags(@allowed_vlan_cache, options)
   end
 
-  def available_vlans(options = {})
+  def available_vlans_and_hosts(options = {})
```

Comment:
This is not my change, I have to cherry pick this from master. This was a fix to a broken provisioning page.